### DISCUSSION TITLE
[Feature] Not only kodi ;-) What about Plex or Jellyfin

### DISCUSSION THREAD

**Discussion by pippo73** [immich-app/immich#14687] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi all, 
I've seen that with the last release [v1.122.3](https://github.com/immich-app/immich/releases/tag/v1.122.3) there is an integration kodi. So I would like to know if there is also the possibility to share photos with Jellyfin or Plex?

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
The Kodi integration isn't an official one, it's just a community project that we added a link to in the docs. Of course, you're more than welcome to make a community integration for Jellyfin or Plex as well ;)

**Comment by cryptonym64** [comment]
The Kodi addon is broken and no sign of being addressed. It used the timeline API against what immich docs say. 

Would really like to see a usable service for DLNA or such, or see real internal development on building clients. As is, a web browser is about all that works.  The Google TV add on also isn't functional for videos especially. Love immich, but lack of integrations is holding it back.
